config/runtimes/kserve-huggingface-tritonserver.yaml

```diff
@@ -0,0 +1,30 @@
+apiVersion: serving.kserve.io/v1alpha1
+kind: ClusterServingRuntime
+metadata:
+  name: kserve-huggingface-tritonserver
+spec:
+  annotations:
+    prometheus.kserve.io/port: '8080'
+    prometheus.kserve.io/path: "/metrics"
+  supportedModelFormats:
+    - name: huggingface
+      version: "1"
+      autoSelect: true
+      priority: 1
```

Comment:
This should be changed to use triton supported model formats

Reply:
currently renamed it to `triton-hf` as this runtime tightly coupled to huggingface